package.json

```diff
@@ -30,6 +30,9 @@
     "uglify-js": "^2.7.5",
     "hyperapp": "^0.12.0"
```

Comment:
Then remove it from here?

Reply:
Wouldn't that break the tests? I see [@hyperapp/router](https://github.com/hyperapp/router/blob/abe55f3dc5ea80af2b47cd33baffdf99203c0a63/package.json#L40) has both.

Reply:
I had figured this out a few weeks ago, but now I can't remember! So, nevermind. 😆